### PR TITLE
VORTEX-6077: Fixing layout of mini style panel.

### PR DESCRIPTION
**Pull request by rkausch**
Fixes #6077

## Proposed Changes
  - Fix layout of mini style panel to avoid horizontal scrolling

### CODE REVIEW

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+    /** The Reset style button. */
+    private IconButton myResetStyleButton;
+
+    /** The Style select combo box. */
+    private final JComboBox<StyleNodeUserObject> myStyleSelectComboBox;
+
+    /** The Style select cb action listener. */
+    private final transient ActionListener myStyleSelectCBActionListener;
+
+    /** The Toolbox. */
```
Comment (Xenotide) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java]: EX: toolbox description doesn't match description in other classes

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+    /** The Style parameter change listener. */
+    private transient VisualizationStyleParameterChangeListener myVisStyleParameterChangeListener;
+
+    /** The Registry change listener. */
+    private transient VisualizationStyleRegistryChangeListener myVisStyleRegistryChangeListener;
+
+    /**
+     * Instantiates a new mini style type panel.
+     *
+     * @param tb the tb
```
Comment (Xenotide) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java]: EX: shorthand parameter names & descriptions, unlike the other classes

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java [hunk 3]
```diff
@@ -0,0 +1,458 @@
...
+        setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        setBorder(BorderFactory.createEtchedBorder());
+        if (selectedNode != null)
+        {
+            handleStyleSelectChange(selectedNode.getStyleClass());
+        }
+    }
+
+    /**
+     * Destroy.
```
Comment (Xenotide) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java]: More description?

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java [hunk 4]
```diff
@@ -0,0 +1,458 @@
...
+     */
+    public boolean isCollapsedPreference()
+    {
+        return myToolbox.getPreferencesRegistry().getPreferences(MiniStylePanel.class).getBoolean(getCollapsedPreferenceKey(),
+                !myIsFirst);
+    }
+
+    /**
+     * Sets the collapsed preference.
+     *
```
Comment (Xenotide) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java]: Extra line.

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanelController.java [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        if (visStyle instanceof AbstractVisualizationStyle && Utilities.sameInstance(visStyle.getClass(), vsClass))
+        {
+            ((AbstractVisualizationStyle)visStyle).setParameters(updateSet, this);
+        }
+    }
+
+    /**
+     * Sets the items.
+     *
+     * @param visualizationControlPanel the f vcp
```
Comment (Xenotide) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanelController.java]: extra space in fvcp

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java (hunk 2 above)
Comment (rkausch) [reply open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java]: fixed

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java (hunk 3 above)
Comment (rkausch) [reply open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java]: fixed

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java (hunk 4 above)
Comment (rkausch) [reply open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java]: fixed

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanelController.java (hunk 5 above)
Comment (rkausch) [reply open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanelController.java]: fixed

**Pull request merged**